circuits/cpp/src/aztec3/circuits/kernel/private/native_private_kernel_circuit_inner.cpp

```diff
@@ -151,10 +134,6 @@ KernelCircuitPublicInputs<NT> native_private_kernel_circuit_inner(DummyComposer&
     // TODO(jeanmon) Resuscitate after issue 499 is fixed as explained below.
     // validate_this_private_call_hash(composer, private_inputs, public_inputs);
 
-    // TODO(rahul) FIXME - https://github.com/AztecProtocol/aztec-packages/issues/499
-    // Noir doesn't have hash index so it can't hash private call stack item correctly
-    // validate_this_private_call_stack(composer, private_inputs);
```

Comment:
Are we sure we should remove this comment? I think we do want to call this helper once the pedersen issue in noir is all set.

Reply:
As mentioned in the other comment, `common_validate_call_stack` does the same thing as `validate_this_private_call_stack` so I removed this.